database/index.js

```diff
@@ -0,0 +1,20 @@
+const mysql = require('mysql');
+const mysqlConfig = require('./config.js');
+
+const connection = mysql.createConnection(mysqlConfig);
+
+const getNeighborhoods = function (callback) {
+  connection.query('SELECT * FROM reviews WHERE neighborhood_id = 4', function (err, response, field) {
+    // console.log('stuff');
```

Comment:
Dont forget to remove console.logs when you can. Your database queries look good

Reply:
Your code looks syntactically correct, the only things that could be improved is a slight code cleanup